__tests__/app.test.js

```diff
@@ -8,10 +8,22 @@ const {
   topicData,
   userData,
 } = require("../db/data/test-data/index");
+const endpoints = require("../endpoints.json");
 
 beforeEach(() => seed({ articleData, commentData, topicData, userData }));
 afterAll(() => db.end());
 
+describe("/api", () => {
+  it("GET: 200 returns an object  ", () => {
```

Comment:
This test description text could be a little more verbose / explicit. What are you expecting to be in the 'object'?